### PR TITLE
Add GSAS tab to Engineering GUI documentation

### LINKED ISSUE #22044: Engineering Diffraction GUI documentation is out of date
[Docs](http://docs.mantidproject.org/nightly/interfaces/Engineering%20Diffraction.html) and [testing docs](http://docs.mantidproject.org/nightly/interfaces/Engineering%20Diffraction%20Test%20Guide.html) for the ED GUI need fixing up a little. They are generally a little out of data, but specifically the testing instructions are faulty (#21861) and the GSAS tab has no documentation.

### PR DESCRIPTION
**Pull request by josephframsay**
Sections on the new GSAS tab were added to the docs for the ED GUI

**To test:**

Build the docs and make sure they look good. Have a read and make sure everything in there sounds sensible/lucid

Fixes #22044 

**Release Notes** 
Doesn't need to be in the release notes, as Engineering scientists aren't using the GUI yet anyway

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- [ ] Is the code of an acceptable quality?
- [ ] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)?
- [ ] Are the unit tests small and test the class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- [ ] Do changes function as described? Add comments below that describe the tests performed?
- [ ] Do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/interfaces/Engineering Diffraction.rst [hunk 1]
```diff
@@ -451,6 +451,69 @@ User may plot single peak fitting workspace in separate window by using
...
+  constants, will probably be of ``.prm`` format
+- **Phase file(s)** - contain crystallographic information about the
+  sample in question. Currently only ``.cif`` files are supported
+
+The following parameters are also required:
+
+- **New GSAS-II Project** - GSASIIRefineFitPeaks creates a new
+  ``.gpx`` project here, which can be opened and inspected from the
+  GSAS-II GUI
+- **GSAS-II Installation Directory** - you must have a version of
```
Comment (DanNixon) [inline docs/source/interfaces/Engineering Diffraction.rst]: Some clarification of what the "installation directory" is may be good. i.e. is it the root of the install, the binary directory, etc.

File: docs/source/interfaces/Engineering Diffraction Test Guide.rst [hunk 2]
```diff
@@ -156,6 +156,44 @@ Negative Testing Ideas
...
+^^^^^^^^^^^^
+
+GSAS-style fitting allows the user to plot peaks from their focused
+NeXuS file (obtained from
+:ref:`focus-Engineering_Diffraction_test-ref`).
+
+After a fit has run, the plot should look something like below:
+
+.. image:: ../images/EnggDiffExampleGSASOutput.png
+    :width: 500px
```
Comment (DanNixon) [inline docs/source/interfaces/Engineering Diffraction Test Guide.rst]: Does RST support percentage specifiers here? For me I have to open the full image to actually read any of the text on it.

File: docs/source/interfaces/Engineering Diffraction Test Guide.rst [hunk 3]
```diff
@@ -156,6 +156,44 @@ Negative Testing Ideas
 - Change any unlocked dialog boxes whilst `Fit` runs
 
 
+.. _gsas-Engineering-Diffraction_test-ref:
+
+GSAS Fitting
```
Comment (DanNixon) [inline docs/source/interfaces/Engineering Diffraction Test Guide.rst]: Is it worth matching the heading with the name of the tab in the GUI (as per the other sections/tabs)?

**Review by DanNixon — CHANGES_REQUESTED** [review]
Just a few comments.

**Review by DanNixon — APPROVED** [review]

**Comment by martyngigg** [comment]
The Ubuntu and Windows jobs both passed but leeroy failed to notify back.

**Pull request merged**